lib/PluginBase.js

```diff
@@ -9,19 +10,11 @@ class PluginBase {
      * Constructor for Plugin class
      * This method is called by js-controller/adapter process internally when initializing the plugin.
      *
-     * @param settings {object} object with configuration:
```

Comment:
This is all now defined in a single file (since the same signature is shared with PluginHandler)